pyvrp/GeneticAlgorithm.py

```diff
@@ -25,6 +25,7 @@ class GeneticAlgorithmParams:
     repair_probability: float = 0.80
     collect_statistics: bool = False
     should_intensify: bool = True
+    nb_iter_no_improvement: int = 10000
```

Comment:
Parameter value we'll need to tune. It was, I think, 20K for DIMACS? (because I'm getting old now, can we write this as `10_000` rather than `10000`?)

Reply:
20K is for DIMACS yes. The 10K is what we used in the competition. Since we'll benchmark on GH1000 instances anyway I think putting it on 20K is a better choice.